test/specs/ellipse.spec.js

```diff
@@ -11,16 +25,111 @@ describe('Ellipse annotation', function() {
     xMax: 4,
     yMax: 4,
     borderWidth: 0
-  };
-
-  window.testEvents(options);
+  }, '(without border)');
 
-  options.rotation = 90;
+  window.testEventsOnBorder({
```

Comment:
so I think this, and the `catch*` and `notCatch*` could be converted to test inRange instead of events.

Reply:
to test inRange and to test an event sounds to me the same thing. Testing an event I can also reuse the same code, otherwise specific functions are needed.

Reply:
I created an example of what I mean: https://github.com/chartjs/chartjs-plugin-annotation/pull/632